<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="6">
     <widget class="QPushButton" name="button1EA5">
      <property name="text">
       <string>1EA5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSpinBox" name="spinAddress">
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>DataNo</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6" colspan="2">
     <widget class="QLabel" name="labelCOM">
      <property name="text">
       <string>COM: Not founded!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineDataNo"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Command</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineCMD"/>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineData"/>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>DataBytes</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="3" colspan="5">
     <widget class="QLineEdit" name="lineAnswer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Answer:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="labelHEX">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineCMD</tabstop>
  <tabstop>lineDataNo</tabstop>
  <tabstop>lineData</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineAnswer</tabstop>
  <tabstop>spinAddress</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
